docs/USAGE.md

```diff
@@ -188,6 +188,41 @@ Example:
 
     OPENVINO_TF_DISABLE_DEASSIGN_CLUSTERS="1"
 
+**OPENVINO_TF_DISABLE_TFFE:**
+Starting from **OpenVINO™ integration with TensorFlow 2.2.0** release, TensorFlow operations are converted by [TensorFlow Frontend](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/tensorflow) to the latest available [Operation Set](https://docs.openvino.ai/latest/openvino_docs_ops_opset.html) by OpenVINO™ toolkit except some exceptional cases. By setting **OPENVINO_TF_DISABLE_TFFE** to **1**, TensorFlow Frontend can be disabled. In that case, TensorFlow Importer (the default translator of **OpenVINO™ integration with TensorFlow 2.1.0** and earlier) will be used to translate TensorFlow operations for all backends. If this environment variable is set to **0**, TensorFlow Frontend will be enabled for all backends. As of **OpenVINO™ integration with TensorFlow 2.2.0** release, this environment variable is effective only on Ubuntu and Windows platforms and TensorFlow Frontend is not supported on MacOS yet. The table below shows the translation modules used for each backend and platform by default for **OpenVINO™ integration with TensorFlow 2.2.0**.
+
+|             | **CPU**     | **GPU**     | **GPU_FP16** | **MYRIAD**  | **VAD-M**   |                                                                |
+|-------------|-------------|-------------|--------------|-------------|-------------|----------------------------------------------------------------|
+| **Ubuntu**  | TF Frontend | TF Frontend | TF Frontend  | TF Importer | TF Importer | _Environment variable changes the default translator_ |
+| **Windows** | TF Frontend | TF Frontend | TF Frontend  | TF Importer | TF Importer | _Environment variable changes the default translator_ |
+| **MacOS**   | TF Importer | TF Importer | TF Importer  | TF Importer | TF Importer | _Environment variable is not effective_                       |
+
+Example:
+
+    OPENVINO_TF_DISABLE_TFFE="1"
+
+**OPENVINO_TF_MODEL_CACHE_DIR:**
+Using this environment variable, a cache directory for [OpenVINO™ model caching](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Model_caching_overview.html). Reusing cached model can reduce the model compile time which impacts the first inference latency using **OpenVINO™ integration with TensorFlow**. Model caching is disabled by default. To enable it, the cache directory should be specified using this environment variable.
```

Comment:
Can we specify that this is an experimental feature?

I think the first sentence needs more clarity. Maybe "Using this environment variable, **you can** create a cache directory" would make more sense.